pkg/utils/utils.go

```diff
@@ -315,6 +315,19 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 			}
 		}
 	}
+	// Set PF link up for kubernetes cluster
+	if ClusterType == ClusterTypeKubernetes {
```

Comment:
Is there a reason this is special for k8s cluster?

Reply:
So we encounter this is k8s vanilla? we can remove this check if you think it make sense openshift.

Reply:
Thanks Moshe!
I think it's fine to enable it for both cluster types, as long as there is numvfs configured for that device, link state will be set to up.
/cc @pliurh

Reply:
In openshift, the NetworkManager will do the job. Shall we check whether the link-state first before setting it to up?

Reply:
There is a check for the state `if pfLink.Attrs().OperState != netlink.OperUp `